docs/content/en/docs/examples/modsecurity.md

```diff
@@ -186,3 +186,24 @@ modsecurity-spoa-6596c6b444-cht27   2/2     Running   0          14m
 modsecurity-spoa-6596c6b444-kw2tr   2/2     Running   0          14m
 modsecurity-spoa-6596c6b444-mkndw   2/2     Running   0          14m
 ```
+
+## Using Coraza instead of ModSecurity
+
+Since the maintainers of ModSecurity are dropping support in 2024, [OWASP has created a replacement called Coraza](https://coreruleset.org/20211222/talking-about-modsecurity-and-the-new-coraza-waf/) which is a drop-in replacement for ModSecurity.
+
+In order to use Coraza, the process is essentially the same as described in the above sections, with a few exceptions. First, in the haproxy-ingress ConfigMap, add the following two additional keys:
+
+```yaml
+  modsecurity-use-coraza: true
+  modsecurity-args: "app=hdr(host) id=unique-id src-ip=src src-port=src_port dst-ip=dst dst-port=dst_port method=method path=path query=query version=req.ver headers=req.hdrs body=req.body"
+```
+
+You may require different `modsecurity-args` depending on your Coraza config and your version of coraza-spoa. Check the [coraza-spoa README](https://github.com/corazawaf/coraza-spoa) for full details.
+
+Second, you'll need to change the container image to a coraza-spoa image (you must supply your own) and create a configmap to hold the Coraza config.yaml. An example can be found [here](/resources/coraza-deployment.yaml).
```

Comment:
missing this file?

Reply:
Turns out the folder was being .gitignored since `resources/` [matches any subfolder](https://git-scm.com/docs/gitignore) named `resources`, including `static/resources`. Fixed in the latested push